src/compass/geo_cslc.py

```diff
@@ -81,8 +81,8 @@ def run(cfg):
                                       image_grid_doppler, ellipsoid, threshold,
                                       iters,
                                       blocksize, dem_margin, flatten,
-                                      azimuth_carrier=az_carrier_poly2d,
-                                      subraster_bounds=burst_bounds)
```

Comment:
Why are we removing the `subraster_bounds` here?

Reply:
I forgot to undo this comment. I was using a different branch and temporarily commented this out in testing.

Reply:
I'll be submitting a PR for GSLC subraster later today so this hopefully this won't be a problem in the near future.